com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/ColladaUtil.java

```diff
@@ -615,10 +615,11 @@ private static ModelImporter.Mesh createModelImporterMesh(List<Float> position_l
                                                               List<Float> texcoord_list,
                                                               List<Float> bone_weights_list,
                                                               List<Integer> bone_indices_list,
-                                                              List<Integer> mesh_index_list) {
+                                                              List<Integer> mesh_index_list,
+                                                              ModelImporter.Material material) {
         ModelImporter.Mesh mesh = new ModelImporter.Mesh();
         mesh.name = "";
-        mesh.material = "";
+        mesh.material = material;
```

Comment:
We've changed from a string (name) to an actual struct, since we want to support material properties soon as well.